lib/rotary-encoder/src/rotary-encoder.cpp

```diff
@@ -41,3 +45,16 @@ ISR(INT0_vect)
     }
     last_trigger_INT0 = t;
 }
+
+volatile unsigned long last_trigger_PCINT0 = 0;
+ISR(PCINT2_vect)
+{
+    unsigned long t = millis();
+    if (t - last_trigger_PCINT0)
+    {
+        if (bit_is_set(PIND, SW_PIN))
+        {
+            button_press();
+        }
+    }
```

Comment:
You forgot to update the time of the last interrupt:

```suggestion
    {
        if (bit_is_set(PIND, SW_PIN))
        {
            button_press();
        }
        last_trigger_PCINT0 = t;
    }
```

Reply:
Addressed in #27